content/applications/general/companies.rst

```diff
@@ -95,41 +202,50 @@ First, make sure each one of your companies is properly set in relation to:
 - :doc:`Fiscal Localizations <../finance/fiscal_localizations>`
 - :doc:`Pricelists <../sales/sales/products_prices/prices/pricing>`
 
-Now, activate the *Inter-Company Transactions* option under *Settings*. With the respective company
-*activated* and *selected*, choose if you would like operations between companies to be synchronized
-at an invoice/bills level or at a sales/purchase orders level.
+Next, navigate to the :guilabel:`Inter-Company Transactions` tab, on the individual company's
+detail form. To do so, go to the :menuselection:`Settings app --> Companies section --> Manage
+Companies`, and select a company from the list.
+
+With the respective company activated and selected, choose one of the following options:
+
+- :guilabel:`Do not synchronize`: do not synchronize any inter-company transactions.
+- :guilabel:`Synchronized invoice/bills`: generates a bill/invoice when a company confirms a
+  bill/invoice for the selected company.
+- :guilabel:`Synchronize Sales Order`: generates a drafted sales order using the selected company
+  warehouse, when a sales order is confirmed for the selected company. If, instead of a drafted
+  sales order, it should be validated, enable :guilabel:`Automatic Validation`.*
+- :guilabel:`Synchronize Purchase Order`: generates a drafted purchase order using the selected
+  company warehouse, when a purchase order is confirmed for the selected company. If, instead of a
+  drafted purchase order, it should be validated, enable :guilabel:`Automatic Validation`.*
+- :guilabel:`Synchronize Sales and Purchase Order`: generates a drafted purchase/sales order using
+  the selected company warehouse, when a sales/purchase order is confirmed for the selected company.
+  If, instead of a drafted purchase/sales order, it should be validated, enable :guilabel:`Automatic
+  Validation`.*
+
+  \*The given option needs to be selected, so :guilabel:`Automatic Validation` appears in the
+  configuration.
```

Comment:
If this notice applies to all the * points, lets break it out from the last bullet point (also add a space).
```suggestion
\* The given option needs to be selected, so :guilabel:`Automatic Validation` appears in the
configuration.
```